vscode-dotnet-runtime-library/src/test/unit/DebianDistroTests.test.ts

```diff
@@ -0,0 +1,185 @@
+/* --------------------------------------------------------------------------------------------
+ *  Licensed to the .NET Foundation under one or more agreements.
+*  The .NET Foundation licenses this file to you under the MIT license.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as chai from 'chai';
+import * as os from 'os';
+import { MockCommandExecutor, MockEventStream } from '../mocks/MockObjects';
+import { DistroVersionPair, DotnetDistroSupportStatus, LinuxVersionResolver } from '../../Acquisition/LinuxVersionResolver';
+import { getMockAcquisitionContext, getMockUtilityContext } from './TestUtility';
+import { DotnetInstallMode } from '../../Acquisition/DotnetInstallMode';
+import { DebianDistroSDKProvider } from '../../Acquisition/DebianDistroSDKProvider';
+const assert = chai.assert;
+const standardTimeoutTime = 100000;
+
+const mockVersion = '7.0.103';
+const acquisitionContext = getMockAcquisitionContext('sdk', mockVersion);
+const mockExecutor = new MockCommandExecutor(acquisitionContext, getMockUtilityContext());
+const pair: DistroVersionPair = { distro: 'Debian', version: '12' };
+const provider: DebianDistroSDKProvider = new DebianDistroSDKProvider(pair, acquisitionContext, getMockUtilityContext(), mockExecutor);
+const shouldRun = os.platform() === 'linux';
+const installType: DotnetInstallMode = 'sdk';
+const noDotnetString = `
+dotnet: command not found
+`
+
+suite('Debian Distro Logic Unit Tests', () =>
+{
+    test('Recommends Correct Version', async () =>
```

Comment:
May you try running these on your machine to see if they pass on debian and providing a screenshot? That'd be helpful.

Reply:
![image](https://github.com/user-attachments/assets/b931d84f-d934-4c78-93a4-b3fd038fcb87)

Reply:
cat /etc/os-release
```cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"```